cmd/integration/integration.go

```diff
@@ -448,29 +451,28 @@ func runSelfLinkTestOnNamespace(c *client.Client, namespace string) {
 }
 
 func runAtomicPutTest(c *client.Client) {
-	var svc api.Service
-	err := c.Post().Resource("services").Body(
-		&api.Service{
-			TypeMeta: api.TypeMeta{
-				APIVersion: latest.Version,
-			},
-			ObjectMeta: api.ObjectMeta{
-				Name: "atomicservice",
-				Labels: map[string]string{
-					"name": "atomicService",
-				},
+	svcBody := api.Service{
+		TypeMeta: api.TypeMeta{
```

Comment:
Why do you set the API version in TypeMeta for some objects, but not for others?